### ISSUE TITLE
Consider joining forces with logisim-evolution?

### ISSUE THREAD

**Issue by willprice** [lawrancej/logisim#90]
Hi,

More of a question than an issue, but are you aware of [logisim-evolution](https://github.com/reds-heig/logisim-evolution)? If so, what are your thoughts, and are there any plans to try and integrate the work done from either side? (i.e. port their work here, or vica versa)

Cheers,

**Comment by lawrancej** [comment]
Indeed, I'd like to work with the folks working on logisim-evolution. I've just been rather busy with other things at the moment. :-/

**Comment by Mechtecs** [comment]
I would not do it. Iam not against the evo project but i think that we cant merge them. This Logisim fork is mainly community driven and the PRs are only checked by lawrancej (Thank you :+1:). The evo fork is more "private" and i cant see any real community.

**Comment by BFH-ktt1** [comment]
I must admit that the development of logisim evolution has been for about 3 years kind of "private" as we tried to merge with Carl Burch. As Carl moved on to other challenges we decided to continue our fork separately from Carl Burch's version, hence the name logisim-evolution was born. The community behind logisim-evolution are three applied universities in Switzerland who actively developed the code-base and added extensive features. We are now actively searching for wider communities to push logisim even further and are of course very interested also in any cooperation with several other forks, like yours, to have even a greater tool than it is already. So we are looking forwards to go in this direction and work with you folks together as time permits (as laweancej already correctly commented, we are all rather busy). This effort, however, will probably be not trivial.
